packages/now-cli/src/util/dev/server.ts

```diff
@@ -550,8 +550,8 @@ export default class DevServer {
       const featHandleMiss = true; // enable for zero config
       const { projectSettings, cleanUrls, trailingSlash } = config;
 
-      const opts = { output: this.output, isBuilds: true };
-      const files = (await getFiles(this.cwd, config, opts)).map(f =>
+      const opts = { output: this.output };
+      const files = (await getFiles(this.cwd, opts)).map(f =>
```

Comment:
What happened to the `config` parameter?

Reply:
The config file was being used for accessing `files: []`, which is no longer around.